dist/dfu.js

```diff
@@ -183,12 +238,14 @@
                 log("found packet characteristic");
                 packetChar = characteristic;
                 service.getCharacteristic(versionUUID)
-                .then(characteristic => {
+                .then(char => { // Older DFU implementations (from older Nordic SDKs) have no DFU Version characteristic. So this may fail.
```

Comment:
I think I changed this name do to my confusion about how scope works in javascript. Will change back.